docs/docs/index.md

```diff
@@ -7,4 +7,14 @@ eleventyNavigation:
 
 Our documentation is hand crafted and optimized to serve as a dictionary to look up details once the need arises.
 
+Go deeper in the following areas:
+
+- [Development](./development/generator/)
```

Comment:
links should always be to `*.md` files so that they work on github and on the website

Reply:
Oh, cool. Maybe an even better reason to have it be generated in some way...maybe not at the 11ty level, but pre-commit style so it can be in GitHub, too? 

BUT, updated, thanks!

Reply:
jup some (pre)processing on files... not sure yet about a good API for it 😅